kentikapi/client.go

```diff
@@ -132,25 +122,34 @@ func (c *Config) FillDefaults() {
 		c.APIURL = APIURLUS
 	}
 
-	if c.CloudExportHostPort == "" {
-		c.CloudExportHostPort = cloudExportHostPort
-	}
-
-	if c.SyntheticsHostPort == "" {
-		c.SyntheticsHostPort = syntheticsHostPort
-	}
-
 	if c.Timeout == nil {
 		c.Timeout = pointer.ToDuration(defaultTimeout)
 	}
 
 	c.RetryCfg.FillDefaults()
 }
 
-func makeConnForGRPC(c Config, hostPort string) (grpc.ClientConnInterface, error) {
+func parseKentikURL(apiURL *string) (string, bool) {
```

Comment:
Is apiURL an output parameter?

Reply:
If input string is a valid url, function return url for grpc resources. If not, return input string directly.